lib/rpmvs.c

```diff
@@ -466,6 +483,13 @@ int rpmvsVerify(struct rpmvs_s *sis, int type,
 	int strength = (sinfo->type | sinfo->strength);
 	int required = 0;
 
+	/* Ignore failure if an alternative exists and verifies ok */
+	if (sinfo->rc == RPMRC_FAIL) {
+	    const struct rpmsinfo_s * alt = getAlt(sis, sinfo);
+	    if (alt && alt->rc == RPMRC_OK)
+		sinfo->rc = RPMRC_NOTFOUND;
+	}
```

Comment:
Why do this, why not just verify all checksums?  This feels prematurely permissive and has the effect of potentially hiding bugs.

If you run `rpm -v --checksig` on an RPM with one incorrect checksum it will tell you that digests are OK and just hide the one that isn't.